pandas/tests/io/parser/conftest.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from pandas import read_csv, read_table
+import pandas.util.testing as tm
+
+
+class BaseParser(object):
```

Comment:
wouldn't these usually go in a base.py or something?  I guess if you need them for the fixtures you could import them from there?

Reply:
`base.py` semantically has generally been used for importing into other test classes.  I'm not sure I see the need to separate out all of configuration so to speak.  The fixtures in `conftest.py` will expose them appropriately once all test classes have been converted.